src/transformers/utils/import_utils.py

```diff
@@ -1387,6 +1387,15 @@ def is_rich_available():
     return _rich_available
 
 
+def check_torch_load_is_safe():
+    if not is_torch_greater_or_equal("2.6"):
+        raise ValueError(
+            "Due to a serious vulnerability issue in `torch.load`, even with `weights_only=True`, we now require to "
+            "upgrade torch to at least v2.6 in order to use the function: `pip install torch --upgrade`"
+            "\nSee the vulnerability report here https://nvd.nist.gov/vuln/detail/CVE-2025-32434"
```

Comment:
```suggestion
            "Due to a serious vulnerability issue in `torch.load`, even with `weights_only=True`, we now require users "
             "to upgrade torch to at least v2.6 in order to load weights in torch .bin format. This version restriction "
             "does not apply when loading safetensors checkpoints."
            "\nSee the vulnerability report here https://nvd.nist.gov/vuln/detail/CVE-2025-32434"
```

Reworded this slightly, and removed the `pip install --upgrade torch` line because this has a chance of breaking users' installs by installing a CUDA-incompatible version!

Reply:
Updated the message, just slightly changed it as it's not necessarily checkpoints that we try to load, so used a more general "files"